app/controllers/api/v2/base_controller.rb

```diff
@@ -144,9 +144,9 @@ def setup_has_many_params
         append_array_of_ids(params)             # unwrapped params
       end
 
-      def check_content_type
```

Comment:
Should this method be deprecated and call the new method to avoid breaking plugins (as we can see is the case for katello)?

Reply:
hmm it's not as easy :thinking: I can do that on method basis, but with the `before_action`, I'll eighter:
- use the old method, but we will trigger our own deprecation callback,
- use new one, but it won't help the katello at all having the old method still around :disappointed:

The renaming is not necessary and I can keep the old name, but I don't like the unability of renaming callback, how should we do it properly?

Reply:
hmm... this is indeed tricky, adding a deprecation will work for `before_action` usage elsewhere, but not for `skip_before_action`.
How about overriding `self.skip_before_action` to rename it if needed and warn of deprecation?

Reply:
Right :+1: I've done that ^_^